app/models/podcast_import.rb

```diff
@@ -6,7 +6,7 @@ class PodcastImport < ApplicationRecord
 
   serialize :config, HashSerializer
 
-  attr_accessor :feed, :feed_raw_doc, :templates
+  attr_accessor :feed, :feed_raw_doc, :templates, :feed_source
```

Comment:
technically unused, but setting up the select dropdown for later when different sources will be implemented.